src/utils/SemaphoreTreeDepthValidator.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+/// @title Semaphore tree depth validator
+/// @author Worldcoin
+/// @notice
+library SemaphoreTreeDepthValidator {
```

Comment:
Have you updated `deploy.js` to work properly with this library?

Reply:
Confirmed